vat_rates.csv

```diff
@@ -316,13 +317,14 @@ UK
 IM",GBP,0.1,standard,
 2011-01-04,,VG,USD,0,standard,British Virgin Islands (British overseas territory) is exempted of VAT.
 2014-01-01,,CP,EUR,0,standard,Clipperton Island (French overseas possession) is exempted of VAT.
+2024-01-01,,CH,CHF,0.081,standard,Switzerland VAT on non-resident providers of digital services to local consumers.
+2018-01-01,2024-01-01,CH,CHF,0.077,standard,Switzerland VAT on non-resident providers of digital services to local consumers.
 2017-01-01,2017-12-31,CH,CHF,0.08,standard,Switzerland VAT on non-resident providers of digital services to local consumers.
-2018-01-01,,CH,CHF,0.077,standard,Switzerland VAT on non-resident providers of digital services to local consumers.
 2017-07-01,,AU,AUD,0.1,standard,Australia VAT on non-resident providers of digital services to local consumers.
 2015-07-01,,KR,KRW,0.1,standard,South Korea VAT on non-resident providers of digital services to local consumers.
 2017-01-01,,RU,RUB,0.18,standard,Russia VAT on non-resident providers of digital services to local consumers.
 2015-10-01,,JP,JPY,0.08,standard,Japan VAT on non-resident providers of digital services to local consumers.
 2014-06-01,,ZA,ZAR,0.14,standard,South Africa VAT on non-resident providers of digital services to local consumers.
 2015-01-01,,AL,ALL,0.2,standard,Albania VAT on non-resident providers of digital services to local consumers.
 2016-10-01,,NZ,NZD,0.15,standard,New Zealand VAT on non-resident providers of digital services to local consumers.
-2011-07-01,,NO,NOK,0.25,standard,Norway VAT on non-resident providers of digital services to local consumers.
+2011-07-01,,NO,NOK,0.25,standard,Norway VAT on non-resident providers of digital services to local consumers.
```

Comment:
@l-naumann Why change this line?

Reply:
I don't quite understand that. I did not knowingly change the line and when I look at the raw version of the file, there isn't any change (also no blank line at the end).

Reply:
@l-naumann I thought it was the newline at the end too, but ok then